File: ObjCExampleWithSwiftPackage/ObjCExampleWithSwiftPackage/ViewController.m
//
//  ViewController.m
//  ObjCExampleWithSwiftPackage
//
//  Created by Meler Paine on 2020/11/9.
//

#import "ViewController.h"
@import SwiftFrameworkExample;
#import "SwiftStaticLibraryExample-Swift.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *titleLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
//  NSString *title = [SwiftFrameworkExample new].title;
  NSString *title = [SwiftStaticLibraryExample new].title;
  self.titleLabel.text = title;
}

@end

File: SwiftExampleWithSwiftPackage/SwiftExampleWithSwiftPackage/ViewController.swift
//
//  ViewController.swift
//  SwiftExampleWithSwiftPackage
//
//  Created by Meler Paine on 2020/11/9.
//

import UIKit
import SwiftFrameworkExample
import SwiftStaticLibraryExample

class ViewController: UIViewController {
  @IBOutlet weak var titleLabel: UILabel!
  
  override func viewDidLoad() {
    super.viewDidLoad()
//    let title = SwiftFrameworkExample().title
    let title = SwiftStaticLibraryExample().title
    titleLabel.text = title
  }
}

